tonytanaka/index.js

```diff
@@ -0,0 +1,145 @@
+// 1
+/*const givenRange = {
+  min: 0,
+  max: 5,
+  check: function (x) {
+    if (x >= givenRange.min && x <= givenRange.max) {
+      return true;
+    }
+    return false;
+  },
+};
+console.log(givenRange.check(6));
+*/
+
+// 1 right way
+function check(num, obj) {
```

Comment:
The name that you used for your function is already a function in JavaScript, that's why you face undefined, try to change it to for example checkIt and it works